src/types.ts

```diff
@@ -146,6 +146,7 @@ export interface Environment {
 
 export interface Error {
   status: 'error'
+  message?: SourceError
```

Comment:
Hmm..would it be possible not to modify the Error type? Since it's used by other components too

Reply:
I think the Context has an array of SourceErrors right? Can we access the SourceError  from the Context?

Reply:
Yeah that works!